cogs/omega.py

```diff
@@ -1,7 +1,123 @@
 import discord
```

Comment:
```suggestion
#!/usr/bin/env python3
# coding: utf-8

import discord
```

Reply:
The `# coding: utf-8` magic comment is useless, because in Python 3 the default source code encoding *is* UTF-8.
For further reading:
https://www.python.org/dev/peps/pep-0263/
https://www.python.org/dev/peps/pep-3120/

The Shebang is also useless in this file, as it is not intended to be executed by a user.
However, it's a good idea to add it in the `main.py` file.